integration-tests/config/periodic/periodic.pict

```diff
@@ -0,0 +1,29 @@
+#
+# This is file used for github periodic job
+#
+
+OCP: 4.19, 4.18, 4.17
+AUTH: github, gitlab
+ACS: new, hosted
+Registry: quay.io, artifactory, nexus
+TPA: new, hosted
+SCM: github, gitlab, bitbucket
+Pipeline: tekton, gitlabci, actions, jenkins
```

Comment:
It needs to align with [tssc-test](https://github.com/redhat-appstudio/tssc-test)
```
ci: CI provider - ["tekton", "jenkins", "gitlabci", "githubactions"]
```